medusa/tv/episode.py

```diff
@@ -273,7 +274,7 @@ def download_subtitles(self, lang=None):
 
             try:
                 notifiers.notify_subtitle_download(self.pretty_name(), subtitle_list)
-            except RequestException as e:
+            except (RequestException, socket.gaierror) as e:
```

Comment:
1. This `try/except` should be removed
2. Add RequestException and gaierror to `exception_handler.py`
3. Delegate to exception_handler.handle(): [here](https://github.com/pymedusa/Medusa/blob/49af1f0a36e3072f0ba468428f07d6c6b72e73fe/medusa/subtitles.py#L1033) and [here](https://github.com/pymedusa/Medusa/blob/49af1f0a36e3072f0ba468428f07d6c6b72e73fe/medusa/subtitles.py#L1038)

That's my opinion